File: python-p/L1-062.py
##L1-062 幸运彩票
'''
2021.11.2
19.10.30
By Xcl
'''
n=int(input())
sum1=0
sum2=0
i=0
j=0
for i in range(n):
        sum1=0
        sum2=0
        str = input()
        ch=[]
        ch.extend(str)
        sum1=int(ch[0])+int(ch[1])+int(ch[2])
        sum2=int(ch[3])+int(ch[4])+int(ch[5])
        if(sum1==sum2):
            print("You are lucky!")
        else:
            print("Wish you good luck.")

File: java-i/L1-062.java
//L1-062 幸运彩票
/*
2021.11.2
18.34.50
By Xcl
*/
import java.util.Scanner;
public class Main
{
    public static void main(String[] args)
    {
	Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        int sum1=0,sum2=0;
        scan.nextLine();//吃掉\n符号;
        for(int i=0;i<n;i++)
        {
        	sum1=0;
        	sum2=0;
            String str = scan.nextLine();
            char[] ch = str.toCharArray();
            for(int j=0;j<ch.length;j++)
            {
                if(j>=0&&j<ch.length-3) sum1 = sum1+ch[j]-'0';
                if(j>=ch.length-3&&j<ch.length) sum2 = sum2+ch[j]-'0';
            }
            if(sum1==sum2) System.out.printf("You are lucky!\n");
            else System.out.printf("Wish you good luck.\n");
        }
    }
}
